src/docs/app-monitoring/sysdig-monitoring-guideline-for-shared-services-and-apps.md

```diff
@@ -0,0 +1,347 @@
+
+## Sysdig dashboard
+One thing that I recommended to start with, is to leverage what someone has already built. We can always use dashboard template that been pre-build by Sysdig dashboard team. And here is a [video demonstrating how to set it up](https://youtu.be/K4rkSCSq3C4?si=cIEqZPnKbgAjwzWE). 
+
+
+
+
+![Sysdig dashboard Flow Chart](../../images/Sysdig-built-in-dashboard.png)
+
+In the dashboard library, you can find dashboards tailored for different purposes. If you wish to edit a dashboard, select it, then click on the __Copy to My Dashboards__ button at the top right to make it your own and modify the queries as needed. If you find a particular dashboard useful, you can click on the __Star__ button in the top right corner. This will save it to your favorites, allowing you to access it quickly in the future.
+
+![Markdown Flow Chart](../../images/favorite-dashboard.png)
+
+## Uptime.com
+
+### Uptime.com: A Brief Overview
+
+[Uptime.com](https://www.uptime.com/) is a comprehensive website monitoring platform. It provides real-time insights into your website's availability, performance, and functionality. By continuously checking websites from multiple locations around the globe, Uptime.com ensures that end-users have the best possible experience.
+
+Key features of Uptime.com include:
+
+- **Availability Monitoring**: Checks if your site is accessible and notifies you immediately if it detects any downtime.
+- **Performance Monitoring**: Measures site speed, helping identify bottlenecks that could impact user experience.
+- **Domain Health Check**: Monitors SSL certificates, domain expirations, and more to ensure the health and security of your domain.
+- **Transactional Tests**: Simulates user paths and interactions on your site, ensuring critical processes like logins or shopping cart checkouts work flawlessly.
+
+By integrating tools like Uptime.com into the SRE toolkit, teams can proactively address issues, ensuring optimal user experience and meeting established Service Level Objectives (SLOs).
+
+We can set up a transactional test for the registry. This test continually loads the dashboard page to ensure all components are successfully loaded. Uptime.com provides insights into the uptime and downtime of your application and can display the percentage of uptime over a specified period. Additionally, it can send out notifications, enabling us to respond promptly.
+
+
+## Runbook
+To achieve 99.5% uptime, we have a daily allowance of just 7 minutes and 12 seconds for downtime. This is where RunBooks prove invaluable. In SRE, the objective is to automate as many processes as feasible. In the realm of cloud operations, Runbooks consist of a series of steps carried out by SREs to accomplish specific tasks. These tasks can encompass incident responses, cost management, addressing performance challenges, and more.
```

Comment:
could you include a brief explanation on runbook, like an automated script that ....